leptonai/cli/photon.py

```diff
@@ -686,62 +677,74 @@ def run(
-                deployment_template=deployment_template,
-                resource_shape=resource_shape,
-                replica_cpu=replica_cpu,
-                replica_memory=replica_memory,
-                replica_accelerator_type=replica_accelerator_type,
-                replica_accelerator_num=replica_accelerator_num,
-                replica_ephemeral_storage_in_gb=replica_ephemeral_storage_in_gb,
-                resource_affinity=resource_affinity,
-                min_replicas=min_replicas,
-                max_replicas=max_replicas,
-                mounts=list(mount),
-                env_list=list(env),
-                secret_list=list(secret),
-                is_public=public,
-                tokens=tokens,
-                no_traffic_timeout=no_traffic_timeout,
-                target_gpu_utilization=target_gpu_utilization,
-                initial_delay_seconds=initial_delay_seconds,
+            spec = LeptonDeploymentUserSpec(
+                photon_id=id,
+                resource_requirement=ResourceRequirement(
+                    resource_shape=resource_shape,
+                    cpu=replica_cpu,
+                    memory=replica_memory,
+                    accelerator_type=replica_accelerator_type,
+                    accelerator_num=replica_accelerator_num,
+                    ephemeral_storage_in_gb=replica_ephemeral_storage_in_gb,
+                    affinity=(
+                        LeptonResourceAffinity(
+                            allowed_dedicated_node_groups=[resource_affinity]
+                        )
+                        if resource_affinity
+                        else None
+                    ),
+                    min_replicas=min_replicas,
+                    max_replicas=max_replicas,
+                ),
+                mount=make_mounts_from_strings(list(mount)),
+                env_list=make_env_vars_from_strings(list(env), list(secret)),
+                api_tokens=make_token_vars_from_config(public, tokens),
+                auto_scaler=AutoScaler(
+                    scale_down=(
+                        ScaleDown(no_traffic_timeout=no_traffic_timeout)
+                        if no_traffic_timeout
+                        else None
+                    ),
+                    target_gpu_utilization_percentage=target_gpu_utilization,
+                ),
+                health=HealthCheck(
+                    liveness=(
+                        HealthCheckLiveness(initial_delay_seconds=initial_delay_seconds)
+                        if initial_delay_seconds
+                        else None
+                    )
+                ),
+            )
+
+            lepton_deployment = LeptonDeployment(
+                metadata=Metadata(id_=id, name=deployment_name), spec=spec
+            )
+            client.deployment.create(lepton_deployment)
+            console.print(
+                f"Photon launched as [green]{deployment_name}[/]. Use `lep deployment"
```

Comment:
Checking back: we missed the second half of the text output f" status -n {deployment_name}` to check the status.". See left.

Reply:
fixing in https://github.com/leptonai/leptonai/pull/434